### PR TITLE
layout student front-end demo

### PR DESCRIPTION
**Pull request by phantuantu24**
tui mới làm cái giao diện thôi...xem v ổn chưa Tuấn Anh

### CODE REVIEW

File: public/stylesheets/style.css [hunk 1]
```diff
@@ -6,3 +6,80 @@ body {
...
+
+/* Add a red background color to the cancel button */
+.form-container .cancel {
+  background-color: red;
+}
+
+/* Add some hover effects to buttons */
+.form-container .btn:hover, .open-button:hover {
+  opacity: 1;
+}
```
Comment (gw2020team5) [inline public/stylesheets/style.css]: - The folder name has been change to "css" so just merge code from "dev" branch

File: views/layout_student.pug [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                $('#chatNow').click(function(){
+                    $('#myForm').show(500);
+                });
+            });
+            function closeForm(){
+                $("#closeF").click(function(){
+                    $('#myForm').hide(500);
+                });
+            }
+    // END CHAT-BOX
```
Comment (gw2020team5) [inline views/layout_student.pug]: This is code to build Messenger so create a new pug file for it then "include" it into layout

File: views/layout_student.pug [hunk 3]
```diff
@@ -0,0 +1,54 @@
+extends layout
```
Comment (gw2020team5) [inline views/layout_student.pug]: This file should be name "role_layout.pug"

File: views/layout_student.pug [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            // Dropdown
+            div(style='position: relative; bottom: 0px; left: 625px;')
+                li.nav-item.dropdown
+                    a#navbardrop.nav-link.dropdown-toggle(href='#' data-toggle='dropdown')
+                        | Welcome [SESSION]
+                    .dropdown-menu
+                        a.dropdown-item(href='#') My Profile
+                        a.dropdown-item(href='#') Change Password
+                        .dropdown-divider
+                        a.dropdown-item(href='#') Logout
```
Comment (gw2020team5) [inline views/layout_student.pug]: We should add a "block" here to make this file become a layout

File: routes/index.js [hunk 5]
```diff
@@ -6,4 +6,8 @@ router.get('/', function(req, res, next) {
   res.render('index', { title: 'eTutor' });
 });
 
+/* GET student layout page. */
+router.get('/layout_student', function(req, res, next) {
+  res.render('layout_student', { title: 'eTutor' });
+});
```
Comment (gw2020team5) [inline routes/index.js]: - Change "/layout_student" to "/student"
- create a new pug file name "dashboard.pug" that extends from "role_layout.pug" below.

File: public/stylesheets/style.css (hunk 1 above)
Comment (trantanhdev) [reply public/stylesheets/style.css]: - should use bootstrap style for button.

File: views/layout_student.pug [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+            li.nav-item
+                a.nav-link(href='#') Blogs
+            // Dropdown
+            div(style='position: relative; bottom: 0px; left: 625px;')
+                li.nav-item.dropdown
+                    a#navbardrop.nav-link.dropdown-toggle(href='#' data-toggle='dropdown')
+                        | Welcome [SESSION]
+                    .dropdown-menu
+                        a.dropdown-item(href='#') My Profile
+                        a.dropdown-item(href='#') Change Password
```
Comment (trantanhdev) [inline views/layout_student.pug]: Comment this code as optional features.

File: views/layout_student.pug [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+        // Brand
+        a.navbar-brand(href='#') Dashboard
+        // Links
+        ul.navbar-nav
+            li.nav-item
+                a.nav-link(href='#') Messages
+            li.nav-item
+                a.nav-link(href='#') Meetings
+            li.nav-item
+                a.nav-link(href='#') Blogs
```
Comment (trantanhdev) [inline views/layout_student.pug]: Should use [pug iteration](https://pugjs.org/language/iteration.html) here, menu items will be a variable list base on user role (tutor, student)

**Pull request closed without merging**